src/Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby ">= 2.6.10"
```

Comment:
Change this back since the latest react native version is still using `>= 2.6.10`.

Reply:
I got an error that this is too old and I had to update it